### ISSUE TITLE
[UI] Better align restart/restage actions with what the CLI allows

### ISSUE THREAD

**Issue by richard-cox** [epinio/ui#200]
- Application restart/restage actions are gated on the application state
- It's been spotted that the CLI allows restage at times when the UI doesn't
- We need to investigate at what stages the CLI allows each action and align the UI to it
  -  It may mean we need to re-evaluate our application state

**Comment by mmartin24** [comment]
@richard-cox, just for you information:

This ticket fixed an issue not applying env vars if not deployed from start: https://github.com/epinio/epinio/issues/2220
This other one will try to include `restart` within  `restage` in the CLI to spare the user from running the 2 commands: https://github.com/epinio/epinio/issues/2230 . I think it would be wise to implement it in the UI as well. Please let me know if you'd prefer a separate ticket for this or would like to consider it within this one.

**Comment by mmartin24** [comment]
Following up work. Restage does now include automatically restarting in cli after the merge of https://github.com/epinio/epinio/pull/2319
